devutils/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "robocorp-devutils"
-version = "0.0.1"
+version = "0.0.2"
```

Comment:
bumping this is not really needed since we don't want to publish ever. you just need to `poetry lock` in the depending package